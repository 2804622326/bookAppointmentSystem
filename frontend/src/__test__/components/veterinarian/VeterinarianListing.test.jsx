import React from 'react';
import { render } from '@testing-library/react';
import '@testing-library/jest-dom';
import VeterinarianListing from '../../../components/veterinarian/VeterinarianListing.jsx';

const vet = { id: 1, firstName: 'A', lastName: 'B', specialization: 'Surg' };

test('renders listing snapshot', () => {
  const { asFragment } = render(<VeterinarianListing veterinarians={[vet]} />);
  expect(asFragment()).toMatchSnapshot();
});
