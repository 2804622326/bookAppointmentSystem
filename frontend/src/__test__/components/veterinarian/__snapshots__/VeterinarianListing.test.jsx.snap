// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders listing snapshot 1`] = `
<DocumentFragment>
  <div>
    <div
      class="d-flex justify-content-center align-items-center mt-5"
      style="height: 100%;"
    >
      <div
        class="spinner-border text-success"
      />
    </div>
  </div>
</DocumentFragment>
`;
